website/docs/r/connect_queue.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Connect"
+layout: "aws"
+page_title: "AWS: aws_connect_queue"
+description: |-
+  Provides details about a specific Amazon Connect Queue
+---
+
+# Resource: aws_connect_queue
+
+Provides an Amazon Connect Queue resource. For more information see
+[Amazon Connect: Getting Started](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-get-started.html)
+
+~> **NOTE:** Due to The behaviour of Amazon Connect you cannot delete queues.
+
+## Example Usage
```

Comment:
I would like to see two more examples that include Caller Confit and Quick Connect Ids for completeness.

Reply:
Example with `quick_connect_ids` resolved in 5942b3be5871a3342640612c1d920efedd1d2a85
Example with `outbound_caller_config` resolved in ebf5333e70f5b1ab524d0d05841c48d61f4003fd